### ISSUE TITLE
Showing error package "cohttp" doesn't exist in the current project for ocaml-cohttp version 1.2.0

### ISSUE THREAD

**Issue by razaulmustafa852** [uccmisl/dashc#6]
I am trying installing DashC from the last 30 days. Every time I faced the same problem as mentioned in the subject or title. You people must provide a clear installation guide. Why cohttp doesn't exist in the current directory. Similarly, a message came, that cohttp pinned to GitHub.
Atleast provide complete commands to run this on Ubuntu 16.04. The guide you provided on GitHub is not enough.

Thanks

**Comment by MMMKassem** [comment]
Have the same problem on Ubuntu 18.04

**Comment by razaunicamp** [comment]
I am installing the dashc inside docker. And docker doesn't work for 18.04
When i tried to install dashc outside docker,
I always face this problem
You need to initialize opam-init first

I really don't know about opam, and this stuff, 
If i install outside docker, how to initialize opam-init, i did with comand as opam-init
But fail

I am more interested to install this inside docker

**Comment by razaulmustafa852** [comment]
Yes I tried again on outside docker. But the same error. As I told. Configure file works, but at the end, the above error came, Later, when I tried to "make". The error came, "The Cohttp doesn't exist in the current project". I installed almost everything but no success. Looking forward to your response.

Package "cohttp" doesn't exist in the current project. (Make Error)
 error 57: Synopsis and description must not be both empty
cohttp-async is now pinned to git+file:///home/raza/Downloads/dashc/ocaml-cohttp#HEAD (version 1.2.0) (Make error)
